tasks/generateFeatureIndex.js

```diff
@@ -0,0 +1,74 @@
+require('colors');
+const glob = require('glob');
+const fs = require('fs');
+const path = require('path');
+
+const FEATURES_PROJECT_DIR = path.join('sauce', 'features');
+const FEATURES_INDEX_PROJECT_PATH = path.join(FEATURES_PROJECT_DIR, 'index.js');
+
+function run(callback) {
+  glob(`${FEATURES_PROJECT_DIR}/*/**/index.js`, (error, files) => {
+    if (error) return callback(error);
+
+    let imports = [];
+    let featureNames = [];
+
+    files.forEach((filePath) => {
+      let filePathSplit = filePath.split(path.sep);
+      let projectFeaturePath = filePathSplit.slice(0, filePathSplit.length - 1).join(path.sep);
```

Comment:
I believe the reason I did it this way is because we need to remove `sauce/features` on line 25 and it was easier with an array